### PR TITLE
fix(condo): DOMA-7829 fix invoice payment export. Fix export receipts payment

### PR DESCRIPTION
**Pull request by VKislov**

### CODE REVIEW

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js [hunk 1]
```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
...
+        await createTestMultiPayment(admin, [invoicePayment], admin.user, acquiringIntegration)
+
+        const {
+            data: {
+                result: {
+                    status,
+                    linkToFile,
+                },
+            },
+        } = await admin.query(EXPORT_PAYMENTS_TO_EXCEL, {
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:111]: export this to test utils

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js [hunk 2]
```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
         })
     })
 
+    it('check payments export', async () => {
+        const locale = 'ru'
+        const { admin, billingReceipts, acquiringContext, acquiringIntegration, organization } = await makePayer()
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:92]: best to export admin and regular data to beforeAll()

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js [hunk 3]
```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
...
+        } = await admin.query(EXPORT_PAYMENTS_TO_EXCEL, {
+            data: {
+                dv: 1,
+                sender: { dv: 1, fingerprint: 'test-' + faker.random.alphaNumeric(8) },
+                where: { invoice: { organization: { id: organization.id } } },
+                sortBy: 'advancedAt_DESC',
+                timeZone: DEFAULT_ORGANIZATION_TIMEZONE,
+            },
+        })
+        const formatDate = (date) => dayjs(date).tz(DEFAULT_ORGANIZATION_TIMEZONE).format('DD.MM.YYYY HH:mm')
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:120]: move to test utils

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js [hunk 4]
```diff
@@ -80,6 +87,100 @@ describe('ExportPaymentsService', () => {
...
+            [
+                formatDate(receiptPayment.advancedAt),
+                receiptPayment.accountNumber,
+                receiptPayment.receipt.property.address,
+                receiptPayment.receipt.account.unitName,
+                acquiringIntegration.name,
+                '',
+                '',
+                i18n('payment.status.' + receiptPayment.status, { locale }),
+                Number(receiptPayment.amount).toFixed(2).toString(),
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:177]: Shouldnt we use bigjs here?

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.js [hunk 5]
```diff
@@ -77,6 +77,7 @@ const ExportPaymentsService = new GQLCustomSchema('ExportPaymentsService', {
 
                 const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
                 const formatDate = (date) => dayjs(date).tz(timeZone).format(DATE_FORMAT)
+                const isInvoicePayments = !!get(where, 'invoice.organization.id')
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.js:80]: best to specify fallback value as something that will be false always, here is an example:

`!!get(where, 'invoice.organization.id', null)`

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.js [hunk 6]
```diff
@@ -93,8 +94,12 @@ const ExportPaymentsService = new GQLCustomSchema('ExportPaymentsService', {
                     return {
                         date: formatDate(obj.advancedAt),
                         account: obj.accountNumber,
-                        address: get(obj, ['receipt', 'property', 'address'], ''),
-                        unitName: get(obj, ['receipt', 'account', 'unitName'], ''),
+                        address: isInvoicePayments ?
```
Comment (toplenboren) [inline apps/condo/domains/acquiring/schema/ExportPaymentsService.js:97]: using giant ternary operators here makes it harder to read this code, please consider exporting it to variables

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js (hunk 4 above)
Comment (VKislov) [reply apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:152]: I think no. I repeated the behavior that is implemented in the export service

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js (hunk 2 above)
Comment (VKislov) [reply apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:92]: I agree, but this utility is used in many places, it’s not something new. Most likely, need to create an admin in `beforAll` and refactor it in the tests that use this utility. I'm not ready to do it in this pull request

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js (hunk 4 above)
Comment (toplenboren) [reply apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:152]: But you are getting amount which is bigJS field i think, number conversions in JS are tricky, why not use something will work correctly 100% time?

I fear that this test will fail randomly from time to time

File: apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js (hunk 2 above)
Comment (toplenboren) [reply apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:92]: On the contrary, this functionality is rather new, and we have agreed that we are going to use beforeAll long before this functionality

Comment (toplenboren) [reply apps/condo/domains/acquiring/schema/ExportPaymentsService.test.js:92]: So I really think that it is best for this PR to implement beforeAll

**Review by toplenboren — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**